src/azure-cli/azure/cli/command_modules/monitor/operations/log_analytics_linked_storage_account.py

```diff
@@ -0,0 +1,34 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+def add_log_analytics_workspace_linked_storage_accounts(client, resource_group_name, workspace_name,
+                                                        data_source_type, storage_account_ids):
+    linked_storage_accounts = client.get(resource_group_name=resource_group_name,
+                                         workspace_name=workspace_name,
+                                         data_source_type=data_source_type)
+    linked_storage_accounts.storage_account_ids.extend(storage_account_ids)
+    return client.create_or_update(resource_group_name=resource_group_name,
+                                   workspace_name=workspace_name,
+                                   data_source_type=data_source_type,
+                                   storage_account_ids=linked_storage_accounts.storage_account_ids)
+
+
+def remove_log_analytics_workspace_linked_storage_accounts(client, resource_group_name, workspace_name,
+                                                           data_source_type, storage_account_ids):
+    linked_storage_accounts = client.get(resource_group_name=resource_group_name,
+                                         workspace_name=workspace_name,
+                                         data_source_type=data_source_type)
+    existed_storage_account_ids_set = set(linked_storage_accounts.storage_account_ids)
```

Comment:
How about write code as follows,

        if storage_account_id.lower() in [existing_storage_acocunt_id.lower() for existing_storage_acocunt_id in linked_storage_accounts.storage_account_ids]:
            linked_storage_accounts.storage_account_ids.remove(storage_account_id)

Reply:
each iteration we need a list generation, which is quite resource consuming. But I change the logic to generate set.